framework/_partials/_preview-splash-page.twig

```diff
@@ -1,5 +1,5 @@
 <!DOCTYPE html>
-<html>
+<html class="no-js">
```

Comment:
Splash page -> you choose the language -> the language is not yet defined. That was my logic